### ISSUE TITLE
queryRunner.dropColumn is ugly

### ISSUE THREAD

**Issue by fwh1990** [typeorm/typeorm#1963]
**Issue type:**

[ ] question

**Database system/driver:**

[ ] `mysql`

**TypeORM version:**

[ ] `@next`

**Steps to reproduce or a small repository showing the problem:**

Why queryRunner.dropColumn(table: string, column: TableColumn)  must input `TableColumn`，Just drop it, as raw sql like:
```sql
ALTER TABLE user DROP COLUMN `name`;
```
It's unnecessary, how do you think? May be like:

```ts
queryRunner.dropColumn(table: string, column: TableColumn | string)
or
queryRunner.dropColumn(table: string, column: string)
```

**Comment by AlexMesser** [comment]
@fwh1990 yes, you're rigth. I already changed this method and it will be available in next release.

**Comment by pleerock** [comment]
Will be released in 0.2.0 very soon.
